documentation/smg/11_ISIS_release.md

```diff
@@ -1,79 +1,161 @@
+Although Horace and Herbert have been merged into a single repository on github, they are stilll represented by two separate repositories on IDAaaS.
+
+#### Safe rapid deploy process
+
+The above deployment method runs some risk of having changes made to the code base which could break on IDAaaS. In particular
+there is no test of the changes on IDAaaS itself. If the change has been made due to an error found by a user on an IDAaaS instrument, it is also
+desirable to have the user test the fix. Consequently alternative repositories are available to isolate changes until they
+are sufficiently tested and approved, and to enable rapid immediate testing on IDAaaS itself. These correspond to the Horace and Herbert repositories 
+in the main deployment process above, and are:
+- Horace_bugfix
+- Herbert_bugfix
+(these have the same underlying and public names.)
+
+The developer should decide whether to make changes on IDAaaS or on their own machine and pass the changes into the IDAaaS repositories.
+The latter case is similar to the main deployment process described above; here we will assume that the changes are being done on IDAaaS
+for immediate consumption.
+- Open a terminal and go to the relevant repositories (Horace_bugfix and/or Herbert_bugfix).
+- check that the repository/ies are checked out to master. If it is otherwise checked out, someone else is doing this process;
+ensure you do not overwrite their changes.
+- Checkout a branch for your ticket and issue e.g. branchname 6666_myname_hotfix_user_issue. You should only need to create the branch in the
+first repository you modify; the other will just need a checkout.
+- make your changes.
+- Open Matlab. It should be initialised with the latest Horace-3 version (3.6.3). To change to the latest Horace-4 with your changes, input
+```
+horace_4on("/usr/local/mprogs/Horace_bugfix")
+```
+- Run such tests as you need to ensure that the user's problem is fixed. The user can carry on working with your bugfix area until github is updated and propagated.
+- As with the main deployment process, now add, commit and push your changes to github, making a pull request.
+- Github via Jenkins should run the tests, and reviewers should approve your changes.
+- when tests pass and reviewers approve, merge your changes into master on github.
+- Now do a git pull on ALL the repositories you have touched; Horace/Herbert to make sure your changes have updated IDAaaS; and Horace_bugfix/Herbert_bugfix
+to ensure that these bugfix areas are ready for the next fix.
+
+#### Stable and historical versions of Horace
+
+The following additional repositories in /usr/local/mprogs hold earlier or current release versions
+- Horace_3_6 (underlying name Horace_3.6): the current stable release version which is being upgraded to Horace-4
+- Horace 3_2 (underlying name Horace; note this is not the same as the symbolically-linked name Horace): preceding historically stable version; not expected to be used
```

Comment:
I've just asked to remove this so, may be removed

Reply:
For 3.2, guessing it's fine if you remove it. I can remove references. For 3.6, i thought that was the default choice if not switched to v4 so checking you're not going to remove that